x-pack/qa/smoke-test-watcher-with-security/src/test/java/org/elasticsearch/smoketest/SmokeTestWatcherWithSecurityIT.java

```diff
@@ -37,6 +37,8 @@
     private static final String TEST_ADMIN_USERNAME = "test_admin";
     private static final String TEST_ADMIN_PASSWORD = "x-pack-test-password";
 
+    private String watchId = randomAlphaOfLength(20);
```

Comment:
Should this be in a `Before` so it's unique per test?

Reply:
I was under the impression, that it is unique per test with this.

```
public class MyTestCase extends ESTestCase {

    private String watchId = randomAlphaOfLength(20);

    public void testFoo() {
        logger.info("id {}", watchId);
    }

    public void testBar() {
        logger.info("id {}", watchId);
    }
}

```

results in

```
[2018-05-02T13:46:10,464][INFO ][o.e.t.MyTestCase         ] [testBar]: before test
[2018-05-02T13:46:10,960][INFO ][o.e.t.MyTestCase         ] id kXWMGSVERErlmIhKUbFd
[2018-05-02T13:46:11,182][INFO ][o.e.t.MyTestCase         ] [testBar]: after test
[2018-05-02T13:46:11,241][INFO ][o.e.t.MyTestCase         ] [testFoo]: before test
[2018-05-02T13:46:11,241][INFO ][o.e.t.MyTestCase         ] id WCGxKOTIEHxFSZIghhBy
[2018-05-02T13:46:11,242][INFO ][o.e.t.MyTestCase         ] [testFoo]: after test
```

Reply:
Sorry, I am not sure why but my mind inserted a `static` into that field definition. That's clearly not there so of course what you have is fine.